docs/samples/python/span_marker_recognizer.py

```diff
@@ -0,0 +1,222 @@
+import logging
+from typing import Optional, List, Dict
+
+from presidio_analyzer import (
+    RecognizerResult,
+    EntityRecognizer,
+    AnalysisExplanation,
+)
+from presidio_analyzer.nlp_engine import NlpArtifacts
+
+try:
+    from span_marker import SpanMarkerModel
+except ImportError:
+    print("Span Marker is not installed")
+
+
+logger = logging.getLogger("presidio-analyzer")
+
+
+class SpanMarkerRecognizer(EntityRecognizer):
+    """
+    Wrapper for a span marker models, if needed to be used within Presidio Analyzer.
+
+    :example:
+    >from presidio_analyzer import AnalyzerEngine, RecognizerRegistry
+
+    >span_marker_recognizer = SpanMarkerRecognizer()
+
+    >registry = RecognizerRegistry()
+    >registry.add_recognizer(span_marker_recognizer)
+
+    >analyzer = AnalyzerEngine(registry=registry)
+
+    >results = analyzer.analyze(
+    >    "My name is Vijay and I live in Pune.",
+    >    language="en",
+    >    return_decision_process=True,
+    >)
+    >for result in results:
+    >    print(result)
+    >    print(result.analysis_explanation)
+
+
+    """
+
+    ENTITIES = [
+        "PERSON",
+        "LOCATION",
+        "ORGANIZATION",
+        # "MISCELLANEOUS"   # - There are no direct correlation with Presidio entities.
+    ]
+
+    DEFAULT_MODEL = "tomaarsen/span-marker-bert-base-fewnerd-fine-super"
+
+    DEFAULT_EXPLANATION = "Identified as {} by Span Marker's Named Entity Recognition"
+
+    PRESIDIO_EQUIVALENCES = {
+        "person-other": "PERSON",
+        "location-GPE": "LOCATION",
+        "organization-company": "ORGANIZATION",
+        "O": "OTHER",  # Outside of a named entity
```

Comment:
Hi @omri374 
Actually if someone wants to use "bert-base-case" typed span marker models, then they need this "O" label as well because they are using this label for label normalization, etc. For now I remove it, if anyone faces that issue they can include this label type and proceed.